docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -86,6 +81,16 @@ like `tctl edit`. With touch required, hardware key support provides better secu
 ## Step 1/2. Enforce hardware key support
 
 Hardware key support is not required by default. 
+
+There are three primary options for hardware key support, in order of security benefit:
```

Comment:
Nit: I would just remove "in order of security benefit" as from the wording it's not clear whether it means increasing or decreasing benefit.
```suggestion
There are three primary options for hardware key support:
```